autoreduce_frontend/autoreduce_webapp/templatetags/natural_time_difference.py

```diff
@@ -4,61 +4,44 @@
 # Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI
 # SPDX - License - Identifier: GPL-3.0-or-later
 # ############################################################################### #
-"""
-Renders the time difference between to given times
-"""
+"""Render the time difference between two given times."""
+# pylint:disable=invalid-name
 from django.template import Library, Node, Variable, TemplateSyntaxError
 from django.template.defaultfilters import pluralize
 
 from autoreduce_frontend.autoreduce_webapp.templatetags.common_helpers import get_var
 
-# pylint:disable=invalid-name
 register = Library()
 
 
 class NaturalTimeDifferenceNode(Node):
-    """
-    Class for computing and rendering time differences
-    """
+    """Class for computing and rendering time differences."""
     def __init__(self, start, end):
         self.start = Variable(start)
         self.end = Variable(end)
 
     def render(self, context):
-        """
-        Render the response
-        """
+        """Render the response."""
         start = get_var(self.start, context)
         end = get_var(self.end, context)
         delta = end - start
         days = delta.days
-        hours, remainder = divmod(delta.seconds, 3600)
-        minutes, remainder = divmod(remainder, 60)
-        seconds = remainder
+        hours = delta.seconds // 3600
+        minutes = delta.seconds // 60 % 60
+        seconds = delta.seconds % 60
+
         human_delta = ''
-        if days > 0:
-            if human_delta:
-                human_delta += ', '
-            human_delta += '%i day%s' % (days, pluralize(days))
-        if hours > 0:
-            if human_delta:
-                human_delta += ', '
-            human_delta += '%i hour%s' % (hours, pluralize(hours))
-        if minutes > 0:
-            if human_delta:
-                human_delta += ', '
-            human_delta += '%i minute%s' % (minutes, pluralize(minutes))
-        if seconds > 0:
-            if human_delta:
-                human_delta += ', '
-            human_delta += '%i second%s' % (seconds, pluralize(seconds))
+        for time, unit in [(days, "day"), (hours, "hour"), (minutes, "minute"), (seconds, "second")]:
+            if time > 0:
+                if human_delta:
+                    human_delta += ', '
+                human_delta += f"{time} {unit}{pluralize(time)}"
```

Comment:
I think this is the coverage line causing the build to fail, though it doesn't look like it's ever been covered. Would you mind adding a test for this (I think it can be in a `autoreduce_webapp/templatetags/test` folder in this PR, or do you think it should be done in a follow up one?